homeassistant/components/homekit_controller/connection.py

```diff
@@ -98,6 +117,14 @@ def __init__(self, hass, config_entry, pairing_data):
 
         self.add_entities()
 
+        await self.async_update()
+
+        async_track_time_interval(
```

Comment:
I notice that you don't track any of the unsub functions for tracking time interval. It is needed to support removing the config entry (but I haven't looked if HomeKit supports that)

Reply:
It currently doesn’t support it, but I’ll keep that in mind when I add support

Reply:
Polling is now turned off when an config entry is unloaded.